src/styles/color-palette.scss

```diff
@@ -1,20 +1,15 @@
-@use 'sass:color';
-@use 'sass:map';
-@function lightenColor($baseColor, $intensity) {
-	$r: color.channel($baseColor, 'red', $space: rgb) +
-		(255 - color.channel($baseColor, 'red', $space: rgb)) * $intensity;
-	$g: color.channel($baseColor, 'green', $space: rgb) +
-		(255 - color.channel($baseColor, 'green', $space: rgb)) * $intensity;
-	$b: color.channel($baseColor, 'blue', $space: rgb) +
-		(255 - color.channel($baseColor, 'blue', $space: rgb)) * $intensity;
+@function lighten($baseColor, $intensity) {
```

Comment:
Copy and paste from https://github.com/gckirchoff/peer-to-public/blob/master/src/styles/color-palette.scss